OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -1,22 +1,24 @@
 #include "TracerThread.h"
 
+#include <linux/perf_event.h>
+
 #include "Logging.h"
 #include "PerfEventProcessor.h"
 #include "PerfEventProcessor2.h"
 #include "PerfEventRingBuffer.h"
 #include "UprobesUnwindingVisitor.h"
 #include "Utils.h"
-#include "absl/container/flat_hash_map.h"
 
 namespace LinuxTracing {
 
+// Record and periodically print basic statistics on the number events.
+constexpr uint64_t EVENT_COUNT_WINDOW_S = 5;
+
 // TODO: Refactor this huge method.
 void TracerThread::Run(
     const std::shared_ptr<std::atomic<bool>>& exit_requested) {
   absl::flat_hash_map<int, PerfEventRingBuffer> fds_to_ring_buffer;
-  absl::flat_hash_map<pid_t, int> threads_to_fd;
```

Comment:
I know that this method is called only once per object, but that's not really enforced, so shouldn't these containers that are now members be cleared here?

Reply:
Or enforce that this is only called once? At least we should document how this is intended to be used.